packages/odyssey-react/src/components/Box/Box.theme.ts

```diff
@@ -31,39 +24,51 @@ export const theme: ThemeReducer = (theme) => ({
   SpaceRemL: theme.SpaceRemL,
   SpaceRemXl: theme.SpaceRemXl,
 
-  // Border
-  BorderWidthBase: theme.BorderWidthBase,
-  BorderStyleBase: theme.BorderStyleBase,
-  BorderRadiusBase: theme.BorderRadiusBase,
-  BorderRadiusOuter: theme.BorderRadiusOuter,
-  ColorBorderBase: "transparent",
-  ColorBorderUi: theme.ColorBorderUi,
-  ColorBorderDisplay: theme.ColorBorderDisplay,
-  ColorBorderPrimary: theme.ColorBorderPrimaryBase,
-  ColorBorderDanger: theme.ColorBorderDangerBase,
+  // Shadow
+  BaseBoxShadow: theme.ShadowBase,
+  HoverBoxShadow: theme.ShadowHover,
 
   // Focus Outline
-  ColorFocusPrimary: theme.ColorFocusPrimary,
-  ColorFocusDanger: theme.ColorFocusDanger,
-  FocusOutlineWidth: theme.FocusOutlineWidthBase,
+  PrimaryFocusBoxShadowColor: theme.ColorFocusPrimary,
+  DangerFocusBoxShadowColor: theme.ColorFocusDanger,
+  FocusBoxShadowSpread: theme.FocusOutlineWidthBase,
+  /* eslint-enable @okta/odyssey/no-invalid-theme-properties */
 
-  // Background
-  ColorBackgroundBase: theme.ColorBackgroundBase,
-  ColorBackgroundDisabled: theme.ColorBackgroundDisabled,
-  ColorBackgroundPrimaryLight: theme.ColorBackgroundPrimaryLight,
-  ColorBackgroundPrimaryBase: theme.ColorBackgroundPrimaryBase,
-  ColorBackgroundPrimaryDark: theme.ColorBackgroundPrimaryDark,
-  ColorBackgroundDangerLight: theme.ColorBackgroundDangerLight,
-  ColorBackgroundDangerBase: theme.ColorBackgroundDangerBase,
-  ColorBackgroundDangerDark: theme.ColorBackgroundDangerDark,
-  ColorBackgroundCautionLight: theme.ColorBackgroundCautionLight,
-  ColorBackgroundCautionBase: theme.ColorBackgroundCautionBase,
-  ColorBackgroundCautionDark: theme.ColorBackgroundCautionDark,
-  ColorBackgroundSuccessLight: theme.ColorBackgroundSuccessLight,
-  ColorBackgroundSuccessBase: theme.ColorBackgroundSuccessBase,
-  ColorBackgroundSuccessDark: theme.ColorBackgroundSuccessDark,
+  // Root
+  FontFamily: theme.FontFamilyBase,
 
-  // Shadow
-  ShadowBase: theme.ShadowBase,
-  ShadowHover: theme.ShadowHover,
+  // Text
+  BaseFontFamily: theme.FontFamilyBase,
+  NormalFontWeight: theme.FontWeightNormal,
+  NormalFontStyle: theme.FontStyleNormal,
+  FontSize: theme.FontSizeBody,
+  BaseLineHeight: theme.FontLineHeightBase,
+  BodyTextColor: theme.ColorTextBody,
+
+  // Border
+  BaseBorderWidth: theme.BorderWidthBase,
+  BaseBorderStyle: theme.BorderStyleBase,
+  BaseBorderRadius: theme.BorderRadiusBase,
+  OuterBorderRadius: theme.BorderRadiusOuter,
+  BaseBorderColor: "transparent",
+  UiBorderColor: theme.ColorBorderUi,
+  DisplayBorderColor: theme.ColorBorderDisplay,
+  PrimaryBorderColor: theme.ColorBorderPrimaryBase,
+  DangerBorderColor: theme.ColorBorderDangerBase,
+
+  // Background
+  BaseBackgroundColor: theme.ColorBackgroundBase,
+  DisabledBackgroundColor: theme.ColorBackgroundDisabled,
+  PrimaryLightBackgroundColor: theme.ColorBackgroundPrimaryLight,
```

Comment:
"LightPrimary" would follow English patterning better, but this allows us to group the related tokens by their main differentiator -- "Primary" tokens are more similar than "Light" tokens.

Reply:
good call out here on this edge case, yea the variants start to get even more numerous when we consider some of the tints/shades. This naming system works just fine for me